addons/ai/fnc_searchNearby.sqf

```diff
@@ -50,6 +50,12 @@ if ((leader _group) distanceSqr _building > 250e3) exitwith {};
 
     // Search while there are still available positions
     private _positions = _building buildingPos -1;
+
+    // Create limiters for each unit
+    private _timeout = (count(_positions) * 15) min 120;
+    private _timetag = "CBASearchTime";
+    {_x setVariable [_timetag,time]} forEach units _group;
```

Comment:
Why not below use getVariable default instead?

Reply:
The units would eventually skip every building as the start time would never be updated. Perhaps the variable name is confusing.

Reply:
Perhaps QGVAR(StartSearchTime) would make more sense.

Reply:
Updated in https://github.com/CBATeam/CBA_A3/pull/1545/commits/aa63e9a12138b64bc8b0826b5dc3fa8447c983ba